### PR TITLE
[downloader] Add option to skip confirmation on cog commands

### PR DESCRIPTION
**Pull request by calebj**
The title says it all. Specifying a third argument recognized by discord.py as a boolean True will skip confirmation for `cog update`, `cog install` and `repo add`.

### CODE REVIEW

**Comment by neonobjclash** [comment]
Bump!

**Comment by calebj** [comment]
It would be nice to get testing and other feedback on this

File: cogs/downloader.py [hunk 1]
```diff
@@ -40,29 +40,30 @@ def save_repos(self):
...
+                               " of being harmful. You will not receive help "
+                               "in Red - Discord Bot #support for any cogs "
+                               "installed from this repo. If you do require "
+                               "support you should contact the owner of this "
+                               "repo.\n\nAgain, ANY repo you add is at YOUR"
+                               " discretion and the creator of Red has "
+                               "ABSOLUTELY ZERO responsibility to help if "
+                               "something goes wrong.")
+            answer = await self.bot.wait_for_message(timeout=15,
+                                                    author=ctx.message.author)
```
Comment (tekulvw) [inline cogs/downloader.py]: PEP8 VIOLATION

**Review by Kowlin — DISMISSED** [review]
Tested and works like a charm for me, if anyone else wants to give it a try. Feel free.

**Comment by tekulvw** [comment]
Is this still valid with the recent downloader updates?

**Comment by Kowlin** [comment]
Currently the PR has a merge conflict that needs to be resolved first.

**Comment by Twentysix26** [comment]
I think this is a fair compromise
https://github.com/Twentysix26/Red-DiscordBot/commit/146d9df460171a41548d204bd3497d666dd99909

**Pull request closed without merging**